test/EmptyApp.FunctionalTests/Startup.cs

```diff
@@ -21,7 +23,9 @@ public class Startup
         // For more information on how to configure your application, visit http://go.microsoft.com/fwlink/?LinkID=398940
         public void ConfigureServices(IServiceCollection services)
         {
-            services.AddInstance<ITelemetryChannel>(new BackTelemetryChannel());
+            services.AddSingleton<IHttpContextAccessor, HttpContextAccessor>();
+            services.AddSingleton<IActionContextAccessor, ActionContextAccessor>();
```

Comment:
Ok apparently ASP.NET doesn't add these, so ignore my comment.

---
In reply to: [59233935](https://github.com/Microsoft/ApplicationInsights-aspnetcore/pull/142#discussion_r59233935) [](ancestors = 59233935)